plugins/dummy/parser/src/dummyparser.cpp

```diff
@@ -17,7 +17,7 @@ DummyParser::DummyParser(ParserContext& ctx_): AbstractParser(ctx_)
 
 bool DummyParser::accept(const std::string& path_)
 {
-  std::string ext = boost::filesystem::extension(path_);
+  auto ext = boost::filesystem::path(path_).extension();
```

Comment:
```suggestion
  auto ext = boost::filesystem::path(path_).extension().string();
```

Reply:
Similarly,  as mentioned in https://github.com/Ericsson/CodeCompass/pull/793#discussion_r2073935439.